utils/zerovec/src/ule/chars.rs

```diff
@@ -40,6 +41,20 @@ use core::convert::TryFrom;
 #[derive(Debug, PartialEq, Eq, Clone, Copy, Hash)]
 pub struct CharULE([u8; 3]);
 
+impl CharULE {
+    /// Converts a [`char`] to a [`CharULE`]. This is equivalent to calling
+    /// [`AsULE::to_unaligned()`]
+    ///
+    /// See the type-level documentation for [`CharULE`] for more information.
+    #[inline]
+    pub const fn from_char(c: char) -> Self {
```

Comment:
Maybe we should call this `to_unaligned_const` so that it has the same name for all types. Wdyt?

Reply:
The reason I didn't go for the `as_ule_const` from the linked issue, is that it seems un-Rusty to me to have a `U::to/as_U(other)` function, but I might not have a good nose for this yet. (See e.g. [here](https://doc.rust-lang.org/std/str/trait.FromStr.html#tymethod.from_str), it's `i32::from_str`, not `i32::to_int(str)`)

I agree that consistent naming would be preferable, do we already have a name for the ULE's AsULE types? Something like `canonical` or `aligned` maybe? Then we could call these functions `from_canonical`.

Reply:
I see what you mean, it should be `char::to_unaligned_const`. We use `aligned`/`unaligned` for `Self` and `Self::ULE`, so I guess that would make it `CharULE::from_aligned`.